### PR TITLE
Add readme for azure spring cloud plugin

### PR DESCRIPTION
**Pull request by andxu**

### CODE REVIEW

File: azure-spring-cloud-maven-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+`includes` | A list of patterns to include, e.g. `**/*.jar`.
+`excludes` | A list of patterns to exclude, e.g. `**/-test.jar`.
+
+Here are a sample configuration:
+
+
+```xml
+<groupId>com.microsoft.azure</groupId>
+<artifactId>azure-spring-maven-plugin</artifactId>
+<version>0.1.0</version>
```
Comment (yucwan) [inline azure-spring-cloud-maven-plugin/README.md]: 1.0.0

File: azure-maven-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - [AzureMaven Plugin](#azure-login-overview)
+  - [Prerequisites](#prerequisites)
+  - [Goals](#goals)
+    - [`azure:login`](#azure-login)
+    - [`azure:select-subscription`](#azure-select-subscription)
+    - [`azure:logout`](#azure-logout)
+
+## Overview
+
+<a name="azure-login-overview"></a>
```
Comment (Flanker32) [inline azure-maven-plugin/README.md]: Do we need this HTML tag?

File: azure-spring-cloud-maven-plugin/README.md (hunk 1 above)
Comment (yucwan) [reply azure-spring-cloud-maven-plugin/README.md]: sample configuration outdated

File: azure-spring-cloud-maven-plugin/README.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+    <deployment>
+        <cpu>1</cpu>
+        <memoryInGB>4</memoryInGB>
+        <instanceCount>1</instanceCount>
+        <runtimeVersion>8</runtimeVersion>
+        <jvmOptions>-XX:+UseG1GC -XX:+UseStringDeduplication</jvmOptions>
+        <environment>
+            <foo>bar</foo>
+        </environment>
+        <volumes>
```
Comment (yucwan) [inline azure-spring-cloud-maven-plugin/README.md]: no more volumes

File: azure-spring-cloud-maven-plugin/README.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+<a name="prerequisites"></a>
+## Prerequisites
+
+<a name="azure-spring-plugin-requirements"></a>
+### System Requirements 
+Tool | Required Version
+---|---
+JDK | 1.8 
+Maven | 3.5.x
```
Comment (yucwan) [inline azure-spring-cloud-maven-plugin/README.md]: Maven 3.0.0

File: azure-maven-plugin/README.md (hunk 2 above)
Comment (andxu) [reply azure-maven-plugin/README.md]: this  link is to define linke for the ### Azure Maven plugin

**Pull request merged**